lambdas/death-enrichment/src/test/java/uk/gov/di/data/lep/DeathEnrichmentIntegrationTest.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.di.data.lep;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.events.SQSEvent;
+import com.amazonaws.services.lambda.runtime.events.SQSEvent.SQSMessage;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.approvaltests.Approvals;
+import org.approvaltests.core.Options;
+import org.approvaltests.scrubbers.GuidScrubber;
+import org.approvaltests.scrubbers.RegExScrubber;
+import org.approvaltests.scrubbers.Scrubbers;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentCaptor;
+import uk.gov.di.data.lep.library.config.Config;
+import uk.gov.di.data.lep.library.dto.GroJsonRecordBuilder;
+import uk.gov.di.data.lep.library.services.AwsService;
+import uk.gov.di.data.lep.library.services.Mapper;
+
+import java.util.List;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+class DeathEnrichmentIntegrationTest {
+    private static final AwsService awsService = mock(AwsService.class);
+    private static final Config config = mock(Config.class);
+    private static final Context context = mock(Context.class);
+    private static final DeathEnrichment underTest = new DeathEnrichment(awsService, config, Mapper.objectMapper());
```

Comment:
It's debatable if this should be in the main test class, but because we mock things like the Object Mapper there, it's tricky to actually test the whole output, so I've spun out this seperate class